setup.cfg

```diff
@@ -0,0 +1,8 @@
+[flake8]
+exclude = *migrations*,
+	  ./tests/*settings*.py,
+	  ./*setup*.py,
```

Comment:
@nemesisdesign Should we make setup.py flake8 compliant or ignore it?

Reply:
why do we need `flake8` and `isort` here? Do we have python code?
Maybe we do, but we don't have any `migrations` or `setup.py` I believe, therefore there's no need to ignore these files if I'm not missing anything.